sample/F8/sponsors

```diff
@@ -0,0 +1,11 @@
+[
+{
+         "id": 1, 
+         "name": "Facebook", 
+         "logo": "", 
+         "level": "1", 
+         "sponsor_type": "Diamond Sponsor", 
+         "url": "www.facebook.com", 
+         "description": ""
+     }, 
```

Comment:
Please set up a gulp/grunt file to pass all json through validators before committing. Thanks :smile: 

This will make it easier for both you and us